ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__gcp-4.16-nightly-x86.yaml

```diff
@@ -35,7 +35,7 @@ tests:
     workflow: openshift-qe-installer-gcp-ipi-ovn-etcd-encryption-fips
   timeout: 5h0m0s
 - as: data-path-etcd-fips-9nodes
-  cron: 0 2 1,15 * *
+  cron: 0 4 1 * *
```

Comment:
Duplicate expression.

I suggest: `0 5 1 * *`

Reply:
same as above, removing 4.12 cron resolved the duplication.

Reply:
I still see a conflict with aws 4.13 data-path.

Should be alright, though. As far as I've seen, multiple data-path jobs don't produce enough data to create contention at indexing time.